week2/2-website/css/style.css

```diff
@@ -15,5 +26,257 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+label {
+  margin: 0 40px 0 0;
+  font-size: 26px;
+  line-height: 70px;
+  display: none;
+}
+#toggle {
+  display: none;
+}
+.all {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.section1 {
+  margin: 0;
+  padding: 0;
+  height: 735px;
+  width: 100%;
+  margin-bottom: 10rem;
+}
+header {
+  background-color: yellowgreen;
+}
+/**************************** The navigator bar style **********************************/
+.nav-bar {
+  position: fixed;
+  display: flex;
+  background-color: white;
+  width: 100%;
+  min-height: 71px;
+}
+/****************************** The container of the list syle************************* */
+.menu-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  width: 60%;
+  height: auto;
+  margin: auto;
+}
+/* ********************************  The logo style ***************************************/
+.logo {
+  width: 3.4%;
+  height: 100%;
+  align-self: center;
+}
+/*********************************** The list style ***************************************/
+.menu {
+  display: flex;
+  background-color: white;
+  justify-content: space-around;
+  list-style-type: none;
+  align-items: center;
+  width: inherit;
+}
+/* ********************************   The links style **************************************/
+.menu li {
+  font-size: 1.3vw;
```

Comment:
> never use `vw` as a unit for font-size or borders.
> This is based on the viewport size so it will shrink to nothing very quickly as we get smaller.
> Better to use `rem`

Done